src/main/java/org/hydev/mcpm/utils/Sugar.java

```diff
@@ -0,0 +1,57 @@
+package org.hydev.mcpm.utils;
+
+import java.util.Arrays;
+
+/**
+ * Syntax sugar that java lacks
+ *
+ * @author Azalea (https://github.com/hykilpikonna)
+ * @since 2022-10-30
+ */
+public class Sugar
```

Comment:
Maybe a different name for this (NegativeSubscriptHelperFriendManFactoryBuilderConstructorPattern)? I was thinking about the cooking ingredient when I saw this lol. Alternatively, do you need the negative subscripting behaviour over copyOfRange? I don't see you using this class anywhere.

Reply:
Lol

I didn't end up using it but it's nice to have in our code